rpa-home/modules/ROOT/pages/invoke-rpa-from-flow-build.adoc

```diff
@@ -0,0 +1,47 @@
+
+# Build an Automation
+
+Implement your automation in RPA Builder.
+
+## Before You Begin
+
+* The following variable types of RPA Builder Activity Parameters are supported as input or output variables of an Invocable Action by Salesforce:
```

Comment:
```suggestion
* Salesforce supports the following variable types of RPA Builder Activity Parameters as invocable action input input or output variables:
```

Reply:
Changed the wording to fit the heading.

Reply:
* For input and output parameters of a process, only use variable types that are supported in Salesforce Invocable Actions as RPA Builder Activity Parameters:
+
The following table shows possible mappings: